src/web/shared/Styles.ts

```diff
@@ -600,10 +600,17 @@ export function getRallyStyle(rallyFeatureCount: number) {
  */
 export function divePacketIconStyle(feature: Feature, animatedColor?: string) {
     // Depth text
-    let text = feature.get('depthAchieved') ? feature.get('depthAchieved').toFixed(1) : null
-    if (text != null) {
-        text = text + 'm'
-    } else {
+    const taskPacket = feature.get('taskPacket') as TaskPacket
+    const dive = taskPacket.dive as DivePacket
+    if (dive == null) {
```

Comment:
We can make this more explicit by saying if (dive !== undefined) since JS handles returns undefined when a property does not exist on an object or we can shorten it to if (dive) {}